build.gradle.kts

```
@@ -110,6 +110,35 @@ githubRelease {
     body = releaseNotes
 }
 
+publishing {
+    publications {
+        create<MavenPublication>("pluginMaven") {
+            pom {
+                name.set("Common Custom User Data Gradle Plugin")
+                description.set("A Gradle Plugin to capture common custom user data used for Gradle Build Scans in Develocity")
```

I would lowercase `plugin` in the description.

```suggestion
                description.set("A Gradle plugin to capture common custom user data used for Gradle Build Scans in Develocity")
```

---

Lowercase plugin in the description